CONTRIBUTING.rst

```diff
@@ -46,6 +67,12 @@ that `pre-commit` may change some of the files to be committed. When this happen
 by re-staging the files, then committing again. This gives you a chance to review the changes brought about by
 `pre-commit`.
 
+Testing Policy
+--------------
+
+The RefRed project make use of automated testing.
+Pull requests should bear this in mind and note that requests that increase the test coverage are heavily preferred.
```

Comment:
Maybe a note should be included here to encourage the developer to test locally at least their changes before submitting the pull request, and preferably the whole test suite.